articles/active-directory/develop/msal-net-client-assertions.md

```diff
@@ -74,33 +74,40 @@ sub | {ClientID} | The "sub" (subject) claim identifies the subject of the JWT,
 Here is an example of how to craft these claims:
 
 ```csharp
-private static IDictionary<string, string> GetClaims()
+using System.Collections.Generic;
+private static IDictionary<string, object> GetClaims(string tenantId, string clientId)
```

Comment:
Generally looks good, and I verified it with one of our integration tests. PR here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3123

I have some comments, mostly minor, except the header which needs a revamp. See the "fixed" version here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/945d9a2f87c70592c54658d456b541463a4bbb3a/tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.cs#L322